dbms/src/main/java/org/frankframework/dbms/DbmsSupportFactory.java

```diff
@@ -47,15 +46,9 @@ public IDbmsSupport getDbmsSupport(DataSource datasource) {
 	}
 
 	private IDbmsSupport compute(DataSource datasource) {
-		try {
-			if (datasource instanceof TransactionalDbmsSupportAwareDataSourceProxy) {
-				Map<String, String> md = ((TransactionalDbmsSupportAwareDataSourceProxy) datasource).getMetaData();
-				return getDbmsSupport(md.get("product"), md.get("product-version"));
-			}
-			try (Connection connection = datasource.getConnection()) {
-				return getDbmsSupport(connection);
-			}
-		} catch (SQLException | DbmsException e) {
```

Comment:
dit moet geloof ik hier ook nog teruggezet worden net als in de 7.9?

Reply:
Nee, dat hoeft niet. In 8.0 en hoger is het blijkbaar anders opgezet dan in 7.9. 
Hier is de enige manier dat deze wordt aangeroepen in productiecode via `JdbcFacade.getDbmsSupport()` die nu de instanceof check doet. In de package `dbms` is dat niet meer mogelijk, omdat de class `TransactionalDbmsSupportAwareDataSourceProxy` is verplaatst naar `core / ... / datasource`.